### ISSUE TITLE
Add worker filtering params to wharf-cmd-provisioner

### ISSUE THREAD

**Issue by jilleJr** [iver-wharf/wharf-cmd#21]
> Depends on #10

## Requirements

- Add query parameters to `GET /api/worker` to add **additional** label filters:
  - `?buildId=123` searches for k8s pod label `wharf.iver.com/build-id=123`
  - `?projectId=123` searches for k8s pod label `wharf.iver.com/project-id=123`

## Out of scope (don't focus on this yet)

- Docker support. Instead focus on Kubernetes
